webpack/components/extensions/HostDetails/Tabs/RepositorySetsTab/RepositorySetsTab.js

```diff
@@ -96,6 +96,25 @@ EnabledIcon.propTypes = {
   isOverridden: PropTypes.bool.isRequired,
 };
 
+const OsRestrictedIcon = ({ osRestricted }) => (
+  <Tooltip
+    position="right"
+    content={__(`OS restricted to ${osRestricted}`)}
+  >
```

Comment:
Translate will be able to handle string interpolation. Need to use <FormattedMessage> for this.

Reply:
An example of usage is here: https://github.com/Katello/katello/blob/master/webpack/components/extensions/HostDetails/Tabs/ModuleStreamsTab/ModuleStreamsTab.js#L121